core/src/syn/parser/stmt/define.rs

```diff
@@ -1212,6 +1216,122 @@ impl Parser<'_> {
 		Ok(res)
 	}
 
+	pub async fn parse_define_config(
+		&mut self,
+		ctx: &mut Stk,
+	) -> ParseResult<DefineConfigStatement> {
+		let (if_not_exists, overwrite) = if self.eat(t!("IF")) {
+			expected!(self, t!("NOT"));
+			expected!(self, t!("EXISTS"));
+			(true, false)
+		} else if self.eat(t!("OVERWRITE")) {
+			(false, true)
+		} else {
+			(false, false)
+		};
+
+		let next = self.next();
+		let inner = match next.kind {
+			t!("GRAPHQL") => {
+				self.pop_peek();
+				self.parse_graphql_config(ctx).await.map(ConfigInner::GraphQL)?
+			}
+			_ => unexpected!(self, next, "a type of config"),
+		};
+
+		Ok(DefineConfigStatement {
+			inner,
+			if_not_exists,
+			overwrite,
+		})
+	}
+
+	pub async fn parse_graphql_config(&mut self, ctx: &mut Stk) -> ParseResult<GraphQLConfig> {
+		use graphql::*;
```

Comment:
Please avoid wild card imports, makes it hard to find to origin of types and has impacts on compile time.